Functions.Templates/templates/TimerTrigger-CSharpWebhookHandler/function.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "bindings": [
+    {
+      "type": "timerTrigger",
+      "direction": "in",
+      "name": "timer",
+      "schedule": "0 15 2 * * *"
+    },
+    {
+      "type": "GraphWebhook",
+      "name": "g",
+      "direction": "in",
+      "PrincipalId": "",
+      "idToken": ""
```

Comment:
PrincipalId and idToken are not present for GraphWebhook binding in bindings.json. Either add them to binding.json or delete from here.